test/loki_service.cc

```diff
@@ -428,7 +440,37 @@ TEST(LokiService, test_osrm_failure_requests) {
   run_requests(osrm_requests, osrm_responses);
 }
 
-// todo: test_success_requests??
+TEST(LokiService, test_actions_whitelist) {
```

Comment:
this is the only real change to the test, a new test that checks getting rejected and checks getting accepted for all actions